Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -27,7 +27,15 @@
 import demistomock as demisto
 import warnings
 
-import signal
+OS_LINUX = False
+OS_MAC = False
+OS_WINDOWS = False
+if sys.platform == "linux" or sys.platform == "linux2":
```

Comment:
use `startswith` instead of `==` as mentioned here https://docs.python.org/3/library/sys.html#sys.platform

Reply:
Done.